### ISSUE TITLE
Failing `xo` linter on master

### ISSUE THREAD

**Issue by jamestalmage** [sindresorhus/generator-nm#24]
With the release of `xo@0.8`, the linter fails on master.

See https://github.com/eslint/eslint/issues/3759

Pinning to `xo@0.7.x` worked on my branch.

**Comment by sindresorhus** [comment]
[No.](https://github.com/eslint/eslint/issues/3759#issuecomment-139714567)

**Comment by jamestalmage** [comment]
But.. Butt..

```
➜  generator-nm git:(master) ✗ npm test

> generator-nm@1.6.2 test /Users/jamestalmage/WebstormProjects/generator-nm
> xo && mocha


/Users/jamestalmage/WebstormProjects/generator-nm/app/index.js
  12:16  error  Properties should be quoted as `default` is a reserved word  quote-props
  36:6   error  Properties should be quoted as `default` is a reserved word  quote-props

✖ 2 problems (2 errors, 0 warnings)
npm ERR! Test failed.  See above for more details.
➜  generator-nm git:(master) ✗ 
```

And that's after an `rm -rf nod_modules` and reinstall.

**Comment by jamestalmage** [comment]
I'm not actually recommending pinning `xo`, just making you aware there is an upstream issue.

**Comment by jamestalmage** [comment]
@ bdebef730619 (1.6.2 release) now fails the build.
 
https://travis-ci.org/jamestalmage/generator-nm/jobs/79963505

**Comment by jamestalmage** [comment]
@sindresorhus see my last comment on the [eslint issue](https://mathiasbynens.be/notes/javascript-properties). It does not appear to be an `eslint` problem. `xo` specifically sets the `keywords: true` options on the `quote-props` rule.

I'll leave it up to you as to whether this is an issue here or with `xo`.

Here is a repro using just pure eslint:

https://github.com/jamestalmage/eslint-3762-repro

If you toggle it to `keywords: false` in package.json the linter passes.

**Comment by sindresorhus** [comment]
@jamestalmage Yes, you're right, sorry about that. The ESLint docs were pretty ambiguous. The `keyword` option is only useful for ES3 as in ES5 those are not reserved in object literals. I removed the `keyword` option in a later XO version. It's fixed in `generator-nm@1.6.3`.
